http4s-scala-fx/src/test/scala/http4s/fx/HttpServerFixtures.scala

```diff
@@ -0,0 +1,38 @@
+package http4s
+package fx
+
+import _root_.fx.{handle, structured, toEither, Control, Resource}
+import _root_.fx.instances.FxAsync.asyncInstance
+import _root_.fx.instances.StructuredF
+import _root_.fx.HttpExecutionException
+import fx.BlazeServerFXBackend
+import munit.FunSuite
+import org.http4s.Method.{GET, POST}
+import org.http4s.{HttpApp, HttpRoutes, Response}
+
+trait HttpServerFixtures:
+  self: FunSuite =>
+
+  val pingPongApp: HttpApp[StructuredF] =
+    HttpApp.apply {
+      case r if r.method == GET && r.uri.path.renderString.contains("ping") =>
+        Response[StructuredF]().withEntity[String]("pong")
+    }
+
+  val echoApp: HttpApp[StructuredF] =
+    HttpApp.apply {
+      case r if r.method == POST && r.uri.path.renderString.contains("echo") =>
+        Response[StructuredF]().withBodyStream(r.body)
+    }
+
+  def httpServerHttp4s(app: HttpApp[StructuredF]): FunFixture[Resource[String]] =
+    FunFixture(
+      setup = _ => {
+        for {
+          server <- toEither(
+            structured(BlazeServerFXBackend(app))
+          ).fold(e => throw e, identity)
+        } yield s"http:/${server.server().address}"
```

Comment:
```suggestion
          server <- BlazeServerFXBackend(app)
        } yield s"http:/${server.server().address}"
```
If we include `Structured` on the context function type contained in the resource above, then we can use the one in the test body surrounding the `resource.use` statement and avoid running the `toEither.fold` here -- allowing any errors in constructing the server builder to be caught by the test framework inside the test instead of by the FunFixture test setup, leading to a clearer test failure and stack trace print by the test reporter, which will be helpful to figure out what is failing.